src/main/java/com/codesquad/rare/domain/post/PostService.java

```diff
@@ -0,0 +1,58 @@
+package com.codesquad.rare.domain.post;
+
+import com.codesquad.rare.domain.account.Account;
+import com.codesquad.rare.domain.account.AccountRepository;
+import com.codesquad.rare.domain.post.request.PostCreateRequest;
+import com.codesquad.rare.domain.post.response.PostCreateResponse;
+import com.codesquad.rare.error.exeception.NotFoundException;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+@Slf4j
+public class PostService {
+
+  private final PostRepository postRepository;
+
+  private final AccountRepository accountRepository;
+
+  public List<Post> findAllAndOrderByCreatedAtDesc(int page, int size) {
+    PageRequest pageRequest = PageRequest.of(page, size, Sort.by("createdAt").descending());
+    return postRepository.findAllByIsPublicTrue(pageRequest).getContent();
+  }
+
+  public Post findById(Long postId) {
+    return postRepository.findById(postId)
+        .orElseThrow(() -> new NotFoundException(Post.class, postId));
+  }
+
+  @Transactional
+  public PostCreateResponse save(PostCreateRequest postCreateRequest) {
+    Account author = accountRepository.findById(postCreateRequest.getAuthorId())
+        .orElseThrow(() -> new NotFoundException(Account.class, postCreateRequest.getAuthorId()));
+    Post savedPost = postRepository.save(Post.toEntity(postCreateRequest, author));
+    PostCreateResponse response = new PostCreateResponse();
+    response.setPostId(savedPost.getId());
+    return response;
+  }
+
+  @Transactional
+  public Post delete(Long postId) {
+    Post post = postRepository.findById(postId)
+        .orElseThrow(() -> new NotFoundException(Post.class, postId));
+    postRepository.delete(post);
+    return post;
+  }
+
+  public List<Post> findAllByLikesInDescendingOrder(int page, int size) {
+    PageRequest pageRequest = PageRequest.of(page, size, Sort.by("likes").descending());
+    return postRepository.findAllByIsPublicTrue(pageRequest).getContent();
```

Comment:
이 메소드가 무엇을 하고 무엇을 리턴하는지 조금 더 명확해지도록 개행을 하면 좋을 것 같습니다.
```suggestion
    return postRepository.findAllByIsPublicTrue(pageRequest)
                          .getContent();
```

Reply:
좋은 방향이네요 ㅎㅎ
@Hamill210 은 개행을 저런 방식으로 하시는 것에 대해 괜찮으신가요?